### PR TITLE
fix(cb2-6062): added default selection to radio group

### PR DESCRIPTION
**Pull request by tom-bjss**
added default value to radio group to reflect new enum as this was based on magic numbers before, this will stop users from being able to press continue without a valid reason selected

### CODE REVIEW

File: src/app/features/tech-record/components/tech-amend-reason/tech-amend-reason.component.ts [hunk 1]
```diff
@@ -23,14 +23,13 @@ export class TechAmendReasonComponent {
 
     this.form = new CustomFormGroup(
       { name: 'reasonForAmend', type: FormNodeTypes.GROUP },
-      { reason: new CustomFormControl({ name: 'reason', type: FormNodeTypes.CONTROL }, 2, [Validators.required]) }
+      { reason: new CustomFormControl({ name: 'reason', type: FormNodeTypes.CONTROL }, 'correcting-an-error', [Validators.required]) }
```
Comment (mcadenas-bjss) [inline src/app/features/tech-record/components/tech-amend-reason/tech-amend-reason.component.ts:26]: If this is to match new enum, would be better to use the enum here when initialising the control value.

**Review by Ian-Hodges — COMMENTED** [review]
Are we okay preselecting an option in a radio group? It's generally something to avoid.
https://design-system.service.gov.uk/components/radios/

**Review by mcadenas-bjss — APPROVED** [review]

**Review by Ian-Hodges — DISMISSED** [review]

**Review by kpearmanbjss — APPROVED** [review]

**Review by aaron-downing — APPROVED** [review]

**Comment by aaron-downing** [comment]
🚀

**Comment by aaron-downing** [comment]
Cosmetic change that's probably not for this PR but would be nice to have some breadcrumbs to be able to navigate to the tech record you were viewing instead of just the home screen or having to click the back button.
image

**Review by mcadenas-bjss — APPROVED** [review]

**Pull request merged**